src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -27,9 +27,7 @@
         "symfony/config": "~2.8|~3.0.0",
         "symfony/console": "~2.7|~3.0.0",
         "symfony/css-selector": "~2.0,>=2.0.5|~3.0.0",
-        "symfony/dependency-injection": "~2.6,>=2.6.6|~3.0.0",
         "symfony/dom-crawler": "~2.0,>=2.0.5|~3.0.0",
-        "symfony/form": "~2.8",
```

Comment:
These two constraints are duplicates of similar ones in framework-bundle, and for some reason (a bug in composer?) they can create [dependency resolving failure](https://travis-ci.org/symfony/symfony/jobs/78934225#L2133). Removing them fixes the issue without changing anything to the resolved set.